packages/duckiebot_interface/launch/all_drivers.launch

```diff
@@ -8,6 +8,9 @@
          doc="The configuration of the robot. For example DB19, DB20, ..."/>
     <arg name="robot_xurdf" value="$(find duckiebot_interface)/urdf/$(arg robot_type).urdf.xacro"
          doc="Location of the URDF XACRO file for the robot type"/>
+    <arg name="dt_sensor_disable_camera" default="0"
+         doc="If 1 the camera node is not started."/>
```

Comment:
This should be
`<arg name="camera_disabled" default="$(optenv DT_SENSOR_CAMERA_DISABLE 0)" doc="Disable camera" />`

Reply:
use exactly what I posted